influxdb/Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: apply
+
+cluster_address:=$(or $(shell PASSWORD_STORE_DIR=~/.registers-pass pass services/influxcloud/admin | tail -n 1), $(error "Couldn't get InfluxCloud cluster address"))
+
+admin_password:=$(or $(shell PASSWORD_STORE_DIR=~/.registers-pass pass services/influxcloud/admin | head -n 1), $(error "Couldn't get InfluxCloud admin password"))
+
+telegraf_password:=$(or $(shell PASSWORD_STORE_DIR=~/.registers-pass pass services/influxcloud/telegraf | head -n 1), $(error "Couldn't get InfluxCloud telegraf password"))
+
+grafana_password:=$(or $(shell PASSWORD_STORE_DIR=~/.registers-pass pass services/influxcloud/grafana | head -n 1), $(error "Couldn't get InfluxCloud grafana password"))
+
+influxdb_configuration.txt: influxdb_configuration.txt.tmpl
```

Comment:
Will this know to regenerate the target if the password changes but there template doesn't?